client/verta/verta/_deployment/endpoint.py

```diff
@@ -124,6 +126,17 @@ def update(self, run, strategy):
         response = _utils.make_request("PUT", url, self._conn, json=strategy._as_build_update_req_body(build_id))
         _utils.raise_for_http_error(response)
 
+        if wait:
+            print("waiting for update...", end='')
+            sys.stdout.flush()
+            while self.get_status()['status'] not in ("active", "error"):
+                print(".", end='')
+                sys.stdout.flush()
+                time.sleep(5)
+            print()
+            if self.get_status()['status'] == "error":
+                raise RuntimeError("endpoint update failed")
+
```

Comment:
Sorry—I know this is a bit outside the scope of this ticket, but I just realized it would make sense to return the endpoint status, similar-ish to what `run.deploy()` does. Can we add this in? Sorrythanks!
```suggestion

        return self.get_status()
```

Reply:
Added. I also updated the tests to use the returned status (instead of calling get_status() again).

Reply:
Awesome, thanks a bunch